web-ui/view/github.ml

```diff
@@ -54,7 +46,11 @@ let history_v ~org ~repo ~history =
     |> List.map @@ fun (commit, status) ->
        li
          ~a:[ a_class [ Build_status.class_name status ] ]
-         [ a ~a:[ a_href (commit_url ~org ~repo commit) ] [ txt commit ] ])
+         [
+           a
+             ~a:[ a_href (Url.commit_url prefix ~org ~repo ~hash:commit) ]
+             [ txt commit ];
+         ])
 
 let link_github_commit ~org ~repo ~hash =
   a ~a:[ a_href (github_commit_url ~org ~repo ~hash) ] [ txt hash ]
```

Comment:
```suggestion
  a ~a:[ a_href (github_commit_url ~org ~repo ~hash) ] [ txt (short_hash hash) ]
```

Reply:
There are a couple of places on the build page and steps page where we still use short hashes. I have line numbers below as I can't seem to make code suggestions outside of already edited blocks:

- Build page, github.ml line 245
- Steps page, github.ml line 317

Change `~hash:(short_hash hash)` argument to `~hash` in both cases.